virtual-host-gatherer/virtual-host-gatherer.spec

```diff
@@ -130,6 +130,11 @@ Nutanix AHV connection module for gatherer
 Summary:        Xen connection module
 Group:          Development/Languages
 Requires:       %{name} = %{version}
+%if 0%{?build_py3}
+Requires:       python3-XenAPI
+%else
+Requires:       python-XenAPI
+%endif
```

Comment:
Just one thing to remember, the `-Xen` subpackage will not be installable until we [package](https://en.opensuse.org/openSUSE:Packaging_Python#The_fast_and_automated_way) [XenAPI](https://pypi.org/project/XenAPI/).